### DISCUSSION TITLE
Alternative to LTMEAT flood for 120.0.6099.272 octopus

### DISCUSSION THREAD

**Discussion by DJshelfmushroom** [3kh0/ext-remover#944] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have a relatively good chromebook, but when I put in the recommended number of pages (2000) it crashes my chromebook.

**Comment by v4pl** [comment]
ya that for me too.

**Comment by AshtonDavies** [comment, marked as answer]
Thanks for letting me know. Most Chromebooks have a memory level of 4, making it 1000 when plugged into the equation. I guess I need to adjust that. For now, decrease your tab count.

Your school is lucky to have nicer than average Chromebooks.

**Reply by v4pl** [reply]
For our chromebooks maybe like 800 tabs or so.

**Reply by v4pl** [reply]
but even with 800 tabs it takes like 1 minute to load and another 1 minute to load the second time.

**Reply by AshtonDavies** [reply]
> but even with 800 tabs it takes like 1 minute to load and another 1 minute to load the second time.

1 minute is normal, and fine. The second time is often a bit faster though.

**Comment by DJshelfmushroom** [comment]
Alr Ill give it some more time

**Reply by CGRmags** [reply]
@DJshelfmushroom if your on 120 do the SKIOVOX 120 and then caub your chromebook in there

**Reply by DJshelfmushroom** [reply]
huh?

**Reply by funkymokey** [reply]
I think he's trying to give advice.

**Reply by DJshelfmushroom** [reply]
yeah but I don't understand what he said

**Reply by DJshelfmushroom** [reply]
skiovox is only for 118 and below

**Reply by AshtonDavies** [reply]
> skiovox is only for 118 and below

There's a 120-121 method.

**Reply by DJshelfmushroom** [reply]
@AshtonDavies where? I don't see it.

**Reply by AshtonDavies** [reply]
> @AshtonDavies where? I don't see it.

It's everywhere dude. Multiple discussions in here, one in TN, another or so in Interstellar.

From here:
https://github.com/3kh0/ext-remover/discussions/900

From my site (if it's better for you, idk):
https://chrose.netlify.app/detail/skiovox/?v=119

**Reply by v4pl** [reply]
Wait Skiovox work for 121 also?

**Reply by AshtonDavies** [reply]
> Wait Skiovox work for 121 also?

Yes, judging by what people are saying if I remember correctly.

**Reply by v4pl** [reply]
wait how tell the method i though it was patched in 121

**Reply by v4pl** [reply]
It's for 120 I did it before but I don't think 121 works if you do know a method plz tell me.

**Reply by DJshelfmushroom** [reply]
oh the website is blocked ):

**Reply by AshtonDavies** [reply]
> oh the website is blocked ):

What website?

**Reply by DJshelfmushroom** [reply]
the netifly tutorial
